package.json

```diff
@@ -48,7 +48,7 @@
     "@babel/preset-react": "^7.0.0",
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
-    "canvas": "^1.6.13",
+    "canvas": "^2.0.0",
```

Comment:
Hi @ntedgi,
Could you downgrade this back to v1.x?
There are some breaking changes and we don't expect them to be included in this life cycle method codemod PR.
https://github.com/Automattic/node-canvas/blob/master/CHANGELOG.md#200

Reply:
hi @chentsulin  thanks for the PR response 
unfortunately i didn't mange to find a way to maintain support for react 17 with that current **canvas** version  , for now i use my fork to remove this  warning . 
if you have any suggestion on how to remove this warning and support  **UNSAFE_**
let me now and i will  be happy to work on it

Reply:
Are you trying react concurrent mode with this module?
It seems that react v17 is not officially released yet.